torch/csrc/Module.cpp

```diff
@@ -175,6 +176,35 @@ static PyObject * THPModule_crashIfATenASAN(PyObject *module, PyObject *arg) {
   return THPUtils_packInt32(at::_crash_if_asan(THPUtils_unpackInt(arg)));
 }
 
+static PyObject * THPModule_vitals_enabled(PyObject *module, PyObject *args) {
+  if (at::vitals::torchVitalEnabled()) {
+    Py_RETURN_TRUE;
+  }
+  Py_RETURN_FALSE;
+}
+
+static PyObject * THPModule_set_vital(PyObject *module, PyObject *args)
+{
+  Py_ssize_t num_args = args ? (Py_ssize_t) PyTuple_Size(args) : 0;
+  THPUtils_assert(num_args == 3, "expected exactly 3 arguments");
+
+  if (!at::vitals::torchVitalEnabled()) {
```

Comment:
Is this needed if `setVital` does its own `torchVitalEnabled()` check?

Reply:
Maybe, kind of, but maybe not? Currently there 3 return values:
- None = vitals not enabled
- True = vitals enabled and value set
- False = vitals enabled and value not set
So technically this is a different return case; but is it helpful to differentiate None and False? Maybe not, since I don't know of anyone who would check it or even under what conditions would cause it other than vitals being disabled. 

Perhaps this does a small amount of optimization by avoiding unpack args when vitals are enabled (reducing overhead)? But the compiler is pretty smart... What do you think?